apps/judicial-system/web/src/types/index.ts

```diff
@@ -16,6 +16,8 @@ export enum LoginErrorCodes {
   UNAUTHENTICATED = 'innskraning-utrunnin',
   LOGIN_FAILED = 'innskraning-ogild',
   DEPRECATED_LOGIN = 'innskraning-gomul',
+  INVALID_USER = 'innskraning-ogildur-notandi',
```

Comment:
Nit: Related to my prev comment, maybe these const could be in the lib so they could be shared between web and backend?

Reply:
Yes they could. A future refactoring task could address this.